client/src/components/ServicesFilter/ServicesFilter.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {listItemSectionBuilder, MultiSelectCheckbox, TableToolbar} from 'wix-style-react';
+import {translate} from '../../utils';
+import {isEqual} from 'lodash';
+
+export default class ServicesFilter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedServices: props.filters.services
+        };
+    }
+
+    _getOptionsList = (options) => {
+        const {servicesGroups} = this.props;
+        for (const serviceGroup in servicesGroups) {
+            if (servicesGroups[serviceGroup].length) {
+                options.push(listItemSectionBuilder({title: translate(`ServicesFilter.${serviceGroup.toLowerCase()}`)}));
+                options.push(...servicesGroups[serviceGroup]);
+            }
+        }
+    };
+
+    _onServiceOptionSelect = (optionScheduleId) => {
+        this.setState({selectedServices: [...this.state.selectedServices || [], optionScheduleId]});
```

Comment:
...this.state.selectedServices || []
should probably be
...(this.state.selectedServices || [])

if you were to have a case were this.state.selectedServices is null or undefined you will crash immediately

Reply:
fixed by initializing filter.services to [] in the store